DataCatalog/examples/0.3/string-db.json

```diff
@@ -1,15 +1,16 @@
 {
   "@context": "http://schema.org",
-  "@type": "WebPage",
-  "@id": "https://string-db.org",
-  "identifier": "stringdb",
+  "@type": "DataCatalog",
+  "@id": "https://string-db.org/",
+  "http://purl.org/dc/terms/conformsTo": "https://bioschemas.org/specifications/DataCatalog/0.3",
```

Comment:
This link https://bioschemas.org/specifications/DataCatalog/0.3 does not work, the link should be https://bioschemas.org/profiles/DataCatalog/0.3-RELEASE-2019_07_01/